profiles-daemon/README.md

```diff
@@ -0,0 +1,261 @@
+       "profile1:MBR10011284",
+       "profile1:MBR5355267"]
+      }
+       ```
+   
+### Run as an Agent service
+
+1. Use the below command to save/create a new agent in wrapping the skill that we created above
+    ```
+       cortex agents save agent.json --project <projectName>
+    ```
+    1. This exposes the profiles-daemon wrapped as a skill as a service, which can be used as the below curl
+   ```
+       curl --location --request POST 'https://api.test.cvstest.gke.insights.ai/fabric/v4/projects/bptest/agentinvoke/realtime/services/input?sync=true' \
+       --header 'Authorization: Bearer '"$CORTEX_TOKEN"'' \
+       --header 'Content-Type: application/json' \
+       --data-raw '{
+        "payload": {
+        "profileSchema": "profile1",
+        "profileId": "MBR5374944"
+        }
+       }
+       '
+   ```
+   
+### Performance numbers
+
+In cluster readings using Jmeter for the agentInvoke APIs with processor-gateway at default scalecount of 3 and profiles-daemon at scalecount of 1 
+
+| Sequence | Concurrency          | 95th Percentile    (Response Time in ms)                                                                                                                                                  | #Samples                                        |
+|----------|----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------|
+| 1        | 1                    | 69.25                                                                                                                                                                                     | 4254                                            |
+| 2        | 10                   | 149.00                                                                                                                                                                                    | 24637                                           |
+| 3        | 20                   | 196.00                                                                                                                                                                                    | 29627                                           |
+| 4        | 40                   | 250.00                                                                                                                                                                                    | 30713                                           |
+| 5        | 80                   | 263.95                                                                                                                                                                                    | 28402                                           |
+
+
+
+
+The APIs we expose:
+* GET request:
```

Comment:
Suggested wording: `The APIs exposed by the Spring application:`

(Not a big difference, but I'm suggesting removing the 'we' because that make the sentence 1st person POV, and I think technical docs are supposed to stay in 2cd person. Might be good to do a quick sweep through the READMEs for this.)